export/cmake/configuration-gen.py

```diff
@@ -0,0 +1,122 @@
+# <<<  User edit
+
+# * For each integrals class, supply a list with max AM for each enabled derivative level.
+# * User is responsible for "distributing" defaults across classes and derivs.
+#   e.g., `--enable-eri3=2 --with-max-am=4` becomes `eri3_max_am = [4, 4, 4]`
+orderings = "ss"
+# max am:
+# * only used for unpaired on eri3.
+# * must be same length as eri3_max_am.
+# * if `--with-max-am` not given to ./configure, duplicate `eri3_max_am` as `max_am` here.
+# * this script will NOT use this as default for other integrals classes.
+max_am = [6, 5]
+multipole = [10]
+onebody = [6, 5, 4]
+eri_max_am = [5, 4]
+eri3_max_am = [6, 5]
+eri3_pure_sh = False
+eri2_max_am = [6, 5]
+eri2_pure_sh = False
+g12_max_am = [4, 4]
+
+# >>>  End user edit
+
+amstr = "SPDFGHIKLMNOPQRTUVWXYZ"
+components = [orderings]
+
+# multipole
+for deriv in range(len(multipole)):
+    for am in range(multipole[deriv], 1, -1):  # don't enumerate s, p
+        centers = amstr[am].lower() * 2
+        comp = f"multipole_{centers}_d{deriv}"
+        components.append(comp)
+
+# onebody
+for deriv in range(len(onebody)):
+    for am in range(onebody[deriv], 1, -1):
+        centers = amstr[am].lower() * 2
+        comp = f"onebody_{centers}_d{deriv}"
+        components.append(comp)
+
+# eri (4-center)
+for deriv in range(len(eri_max_am)):
+    for am in range(eri_max_am[deriv], 1, -1):
+        centers = amstr[am].lower() * 4
+        comp = f"eri_{centers}_d{deriv}"
+        components.append(comp)
+
+# eri3
+no_pure_sh = []
+for deriv in range(len(eri3_max_am)):
+    for am in range(eri3_max_am[deriv], 1, -1):
+        for am2 in range(max_am[deriv], 1, -1):
+            if am2 >= am:
+                centers = amstr[am].lower() * 2 + amstr[am2].upper()
```

Comment:
I need to switch these to `{max_am}{max_am}{eri3_max_am}`. Apparently I had it right in my head (from you) when writing INSTALL.md but switched it around by the time I wrote the script.

Reply:
fixed in 3171be8